tox.ini

```diff
@@ -1,15 +1,16 @@
 [tox]
-envlist = py27,py33,py34,py35,pypy,pypy3,pep8
+envlist = py37,py38,pypy3,pep8
 
 [testenv]
 deps = -e{toxinidir}[test]
-commands = py.test {posargs}
+usedevelop = True
```

Comment:
Was the `usedevelop = True` necessary to run the tests? If so - can you explain the reasoning behind? I am curious.

Reply:
Yes it was and it looks like it fixed #12.

Honestly, I did not had the time yet to completely understand why it behaved like that in this package while not in others. 

I have taken a look at several other package from the project and saw that in morepath/reg> use it. I understand what it does but I am not completely sure why it is needed here yet.